client/state/notices/middleware.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { truncate } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { successNotice, errorNotice } from 'state/notices/actions';
+import { getSitePost } from 'state/posts/selectors';
+import {
+	POST_DELETE_FAILURE,
+	POST_DELETE_SUCCESS,
+	POST_SAVE_SUCCESS
+} from 'state/action-types';
+
+export const OBSERVERS = {
```

Comment:
>This feels qualitatively different to me [...]

Yeah, on considering this some more, I'm not sure the all-caps is appropriate here. To make matters more complicated, we (and be "we", I mostly mean "I" 😄 ) have used `PascalCase` in some instances of objects consisting only of constants ([example](https://github.com/Automattic/wp-calypso/blob/4d92cde41ca1ecc7bcf4e7809b7562a87c785b96/client/lib/media/constants.js#L12-L20)), which is already at odds with how I've implemented it here. I can't recall where specifically, but this came up as a recent discussion with @nylen on whether this was a good approach (and should be documented).

Do you think this is a distinction worth making (`Handlers`) or should we keep to our current guidelines and name as lowercase (`handlers`)?